plugins/de.cognicrypt.codegenerator.tests/src/de/cognicrypt/codegenerator/generator/test/EncryptionCodeGenTest.java

```diff
@@ -0,0 +1,116 @@
+package de.cognicrypt.codegenerator.generator.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.ICompilationUnit;
+import org.eclipse.jdt.core.IJavaProject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.cognicrypt.codegenerator.generator.CodeGenerator;
+import de.cognicrypt.codegenerator.generator.CrySLBasedCodeGenerator;
+import de.cognicrypt.codegenerator.tasks.Task;
+
+import de.cognicrypt.codegenerator.testutilities.TestUtils;
+import de.cognicrypt.codegenerator.generator.test.Constants;
+import de.cognicrypt.codegenerator.wizard.Configuration;
+import de.cognicrypt.utils.DeveloperProject;
+
+/**
+ * @author Shahrzad Asghari
+ */
+public class EncryptionCodeGenTest {
+	
+	Logger log = Logger.getLogger(EncryptionCodeGenTest.class.getName());
+	IJavaProject testJavaProject;
+	CodeGenerator generatorEnc;
+	Task encTask;
+	Configuration configEnc;
+	DeveloperProject developerProject;
+	IResource targetFile;
+	ICompilationUnit testClassUnit;
+	
+	@After
+	public void tearDown() throws CoreException {
+		TestUtils.deleteProject(this.testJavaProject.getProject());
+	}
+
+	@Before
+	public void setUp() throws Exception {
+		this.testJavaProject = TestUtils.createJavaProject(Constants.PROJECT_NAME);
+		targetFile = TestUtils.generateJavaClassInJavaProject(this.testJavaProject, Constants.PACKAGE_NAME, Constants.CLASS_NAME);
+		this.encTask = TestUtils.getTask("Encryption");
+		this.generatorEnc = new CrySLBasedCodeGenerator(targetFile);
+		this.developerProject = this.generatorEnc.getDeveloperProject();
+		this.testClassUnit = TestUtils.getICompilationUnit(this.developerProject, Constants.PACKAGE_NAME, Constants.JAVA_CLASS_NAME);
+		TestUtils.openJavaFileInWorkspace(this.developerProject, "testPackage", this.testClassUnit);
+
+	}
+	@Test
+	public void testCodeGenerationEncryption() throws CoreException, IOException {
+
```

Comment:
Please fix formatting in all classes: No empty lines in the beginning or at the end of methods.